Makefile.core.mk

```diff
@@ -344,6 +345,7 @@ lint: lint-python lint-copyright-banner lint-scripts lint-go lint-dockerfiles li
 go-gen:
 	@mkdir -p /tmp/bin
 	@go build -o /tmp/bin/mixgen "${REPO_ROOT}/mixer/tools/mixgen/main.go"
+	@go build -o /tmp/bin/featuresgen "${REPO_ROOT}/test/framework/tools/featuresgen/main.go"
```

Comment:
Can we just use the `go generate ./...` call on l349 to cover this?

Reply:
perhaps we run featuresgen directly from go without compiling?  @howardjohn is that what you mean?

Reply:
We have tons of generation code, we can do the same as we do for those. for example `galley/pkg/config/testing/basicmeta/generate.go`. no need to add more complexity